content/docs/learning-guide/tutorials/ai/_index.md

```diff
@@ -1,15 +1,13 @@
 ---
-linkTitle: Kythera AI
-title: Using Kythera AI in Open 3D Engine
-description: The Kythera AI Gem provides support for Kythera AI features in Open 3D Engine (O3DE), and includes a demo project, levels, and assets that demonstrate the features of Kythera AI.
+linkTitle: AI
+title: Artificial Intelligence (AI) Tutorials
+description: Learn to create artificial intelligence (AI) for your Open 3D Engine (O3DE) projects.
 weight: 100
```

Comment:
I suggest we remove the `weight` front matter from all tutorial _index.md files, to let them automatically sort in alphabetical order.
```suggestion
```